docs/manual/scala/releases/Migration14.md

```diff
@@ -59,7 +69,7 @@ The change to drop support for multiple services will require another code updat
   )
 ```
 
-That is now deprecated and will issue a warning on runtime (unfortunately Scala 2.11 will not cause a compilation warning because of the deprecation). The method replacing `describeServices` is `describeService` (in singular) and it will take an `Option[Descriptor]` instead of a list:
+That is now deprecated and will issue a warning on runtime (unfortunately Scala will not cause a compilation warning because of the deprecation). The method replacing `describeServices` is `describeService` (in singular) and it will take an `Option[Descriptor]` instead of a list:
```

Comment:
As an aside, Scala 2.12 has a [`deprecatedOverriding`](https://www.scala-lang.org/api/2.12.2/scala/deprecatedOverriding.html) annotation we can use in the future. However, despite the scaladoc saying it is available since 2.10, it was `private[scala]` until 2.12, so we can't use it until we drop 2.11 support.

It remains to be seen whether we drop 2.11 support before we drop this deprecated method 😄